src/App.tsx

```diff
@@ -1,25 +1,88 @@
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [userValue, setUserValue] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const idCreator = () => todosFromServer.length + 1;
+
+  const createToDo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!todoTitle) {
+      setTitleError(true);
+
+      return;
+    }
+
+    if (!userValue) {
+      setUserError(true);
+
+      return;
+    }
+
+    todosFromServer.push({
```

Comment:
Directly modifying the data from the server is not a good practice. Instead, maintain a separate state for the todos in the component.